test/standalone/test-sourcemapper.js

```diff
@@ -0,0 +1,417 @@
+};
+
+testTool('Babel', BABEL_SOURCEMAP, [
+  [1, 14],
+  [2, 15],
+  [3, 16],
+  [4, 17],
+  [5, 18],
+  [6, 19],
+  [8, 21],
+  [9, 22],
+  [11, 24],
+  [12, 26],
+  [13, 27],
+  [14, 30],
+  [15, 31],
+  [16, 32],
+  [18, 36],
+  [19, 37],
+  [20, 38],
+  [21, 39],
+  [23, 42],
+  [24, 43],
+  [25, 44],
+  [28, 50],
+  [29, 53],
+  [30, 56],
+  [31, 58],
+  [32, 60],
+  [34, 64],
+  [35, 65],
+  [36, 66],
+  [37, 67],
+  [39, 70],
+  [40, 71],
+  [44, 78],
+  [45, 81],
+  [47, 83],
+  [50, 85],
+  [54, 88],
+  [55, 89],
+  [56, 90],
+  [59, 93],
+  [62, 99],
+  [63, 102],
+  [66, 105],
+  [69, 108],
+  [70, 109],
+  [73, 112],
+  [74, 113],
+  [77, 116],
+  [78, 117],
+  [79, 118]
+]);
+
+// see the comments below for details on the code and tool used to 
+// generate this sourcemap
+var TYPESCRIPT_SOURCEMAP = {
+  'version': 3,
+  'file': 'out.js',
+  'sourceRoot': '',
+  'sources': [
+    'in.ts'
+  ],
+  'names':[],
+  'mappings': ';;;;;AACA;IACE,gBAAmB,IAAY,EAAS,KAAa;QAAlC,SAAI,GAAJ,IAAI,'+
+    'CAAQ;QAAS,UAAK,GAAL,KAAK,CAAQ;QACnD,IAAI,CAAC,IAAI,GAAG,IAAI,CAAC;'+
+    'QACjB,IAAI,CAAC,KAAK,GAAG,KAAK,CAAC;IACrB,CAAC;IAED,sBAAK,GAAL;QACE,'+
+    'MAAM,CAAC,MAAM,GAAG,IAAI,CAAC,IAAI,GAAG,QAAQ,GAAG,IAAI,CAAC,KAAK,CAAC;'+
+    'IACpD,CAAC;IACH,aAAC;AAAD,CAAC,AATD,IASC;AAED;IAAmB,wBAAM;IACvB;QACE,'+
+    'kBAAM,MAAM,EAAE,MAAM,CAAC,CAAC;IACxB,CAAC;IACH,WAAC;AAAD,CAAC,AAJD,'+
+    'CAAmB,MAAM,GAIxB;AAED,eAAe,IAAY;IACzB,MAAM,CAAC,QAAQ,GAAG,IAAI,CAAC;'+
+    'AACzB,CAAC;AAED,KAAK,CAAC,KAAK,CAAC,CAAC;AACb,IAAI,IAAI,EAAE,CAAC,'+
+    'KAAK,EAAE,CAAC'
+};
+
+testTool('Typescript', TYPESCRIPT_SOURCEMAP, [
+  [1, 5],
+  [2, 6],
+  [3, 9],
+  [4, 10],
+  [7, 12],
+  [8, 13],
+  [12, 17],
+  [13, 19],
+  [14, 20],
+  [18, 24],
+  [19, 25],
+  [22, 27],
+  [23, 28]
+]);
+
+// see the comments below for details on the code and tool used to 
+// generate this sourcemap
+var COFFEESCRIPT_SOURCEMAP = {
+  'version': 3,
+  'file': 'in.js',
+  'sourceRoot': '',
+  'sources': [
+    'in.coffee'
+  ],
+  'names': [],
+  'mappings': ';AACA;AAAA,MAAA,iDAAA;IAAA;;;EAAM;IACS,gBAAC,IAAD,EAAO,KAAP;'+
+    'MACX,IAAC,CAAA,IAAD,GAAQ;MACR,IAAC,CAAA,KAAD,GAAS;IAFE;;qBAIb,KAAA,GAAO,'+
+    'SAAA;AACL,aAAO,MAAA,GAAS,IAAT,GAAgB,QAAhB,GAA2B;IAD7B;;;;;;EAGH;;;IACS,'+
+    'cAAA;MACX,sCAAM,MAAN,EAAc,MAAd;IADW;;;;KADI;;EAInB,MAAA,GACE;IAAA,CAAA,'+
+    'EAAI,GAAJ;IACA,CAAA,EAAI,GADJ;;;EAGF,OAAO,CAAC,GAAR,CAAY,IAAI,IAAA,CAAA,'+
+    'CAAM,CAAC,KAAvB;;EAEA,IAAA,GAAO,CAAC,CAAD,EAAI,CAAJ,EAAO,CAAP,EAAU,CAAV,'+
+    'EAAa,CAAb;;EACP,MAAA,GAAS,SAAC,CAAD;WAAO,CAAA,GAAE;EAAT;;EACT,UAAA;;'+
+    'AAAc;SAAA,sCAAA;;mBAAA,MAAA,CAAO,CAAP;AAAA;;;AApBd'
+};
+
+testTool('Coffeescript', COFFEESCRIPT_SOURCEMAP, [
+  [1, 1],
+  [2, 7],
+  [3, 8],
+  [4, 9],
+  [6, 12],
+  [7, 13],
+  [9, 20],
+  [10, 23],
+  [11, 24],
+  [13, 31],
+//  [14, 32],
+  [15, 33],
+  [17, 36],
+  [19, 38],
+  [20, 40],
+  [21, 44]
+]);
+
+// ---------------------- supplemental information ------------------------
+
+/*
```

Comment:
I've made a new `sourcemapper` directory under `fixtures` to hold the input, output, build, and sourcemap files.